makefiles/common.mk

```diff
@@ -0,0 +1,16 @@
+PROJECT=lfs-s3
+GO_FILE=$(shell find . -type f -name "*.go")
+TAG=$(shell git describe --tags | sed 's|^v||' | sed 's|\(\.*\)-.*|\1|')
```

Comment:
Determine the version based off of git tag.  Converts `v1.2.3` into `1.2.3`

Reply:
Could be a single sed command, but that's still fine.